src/main/java/friday/Friday.java

```diff
@@ -0,0 +1,45 @@
+package friday;
+
+import java.util.Scanner;
+
+/**
+ * Represents the main class for the Friday application.
+ */
+public class Friday {
+
+    private TaskList taskList;
+    private Storage storage;
+    private Scanner input;
+    private Ui ui;
+    private Parser parser;
+
+    /**
+     * Constructs a new instance of the Friday application.
+     * Initialises the user interface, task list, input scanner, storage, and parser.
+     *
+     * @param filePath The path to the storage file.
+     */
+    public Friday(String filePath) {
+        this.ui = new Ui();
+        this.taskList = new TaskList();
+        this.input = new Scanner(System.in);
+        this.storage = new Storage(filePath);
+        this.parser = new Parser();
+    }
+
+    public static void main(String[] args) {
+        Friday friday = new Friday("data/tasks.txt");
```

Comment:
Should this be declared above as a `private static final String` constant?

```suggestion
        Friday friday = new Friday(Friday.DATA_FILE_PATH);
```